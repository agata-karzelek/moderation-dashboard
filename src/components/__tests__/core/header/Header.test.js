import React from "react";
import { shallow } from "enzyme";
import Header from "../../../core/header/Header";

jest.mock("react-router-dom", () => ({
  useHistory: jest.fn().mockReturnValue({
    length: 8,
    action: "PUSH",
    location: {
      pathname: "/overview",
      search: "",
      hash: "",
      key: "t2wbs9",
    },
  }),
}));

describe("Test Cases for Header", () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Header />);
  });

  test("Snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
