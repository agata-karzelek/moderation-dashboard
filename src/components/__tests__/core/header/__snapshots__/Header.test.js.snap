// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Cases for Header Snapshot 1`] = `
<div
  className="makeStyles-grow-1"
>
  <WithStyles(ForwardRef(AppBar))
    className="makeStyles-root-2"
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <WithStyles(ForwardRef(IconButton))
        aria-label="open drawer"
        color="inherit"
        edge="start"
        id="toDashboard"
        onClick={[Function]}
      >
        <img
          alt=""
          src="/images/Pubnub logo.svg"
          width={90}
        />
      </WithStyles(ForwardRef(IconButton))>
      <div
        className="makeStyles-verticalLine-5"
      />
      <Breadcrumb
        currentPage=""
      />
      <div
        className="makeStyles-grow-1"
      />
      <WithStyles(ForwardRef(Grid))
        item={true}
      >
        <WithStyles(ForwardRef(Button))
          className="makeStyles-button-4"
        >
          Docs
        </WithStyles(ForwardRef(Button))>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        item={true}
      >
        <div
          className="makeStyles-verticalLine-5"
        />
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        className="makeStyles-headerMenu-3"
        item={true}
      >
        <MyAccountDropdown
          accounts={Array []}
        />
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
